docs/zh/UserGuide/Maintenance-Tools/Backup-Tool.md

```diff
@@ -0,0 +1,61 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+    
+        http://www.apache.org/licenses/LICENSE-2.0
+    
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+# Backup-Tool
+IOTDB 的数据导入导出工具,导出的数据主要包含时间序列的结构和其对应的数据。
```

Comment:
When adding new pages, remember to update the site/src/main/.vuepress in master branch.

Reply:
Does that mean I need to submit a special pr to do these changes？